cypress/integration/vote_for_card_spec.js

```diff
@@ -0,0 +1,58 @@
+describe('Vote for a card', () => {
+    beforeEach(() => {
+        cy.login('unicorn');
+        /* Connect a second user using our fake client */
+        cy
+            .request(`http://localhost:12346/connect?url=${encodeURIComponent(Cypress.config().baseUrl)}`)
+            .then((response) => {
+                if (response.status === 200) {
+                    cy.startGame();
+                    cy.request(`http://localhost:12346/playCardWord?url=${encodeURIComponent(Cypress.config().baseUrl)}`)
+                }
+            });
+        cy.get('#current-player').then(($player) => {
+            if ($player.text() === 'Current player: unicorn') {
+                cy.get('[data-cy="type-word"]').type('word');
+                cy.get('[data-cy="send-word"]').click();
+                cy.get('[data-cy="my-cards"] > img').first().click();
+            } else {
+                cy.get('[data-cy="my-cards"] > img').first().click(); 
+            }
+        });
+    });
+
+    describe('on player plays a card and then votes for one', () => {
+        it('displays all played cards after current player plays their card', () => {
+            cy.get('[data-cy="played-cards"]').children().its('length').should('eq', 2);
+        });
+        it('player is prompted to vote for a card', () => {
+            cy.get('[data-cy="vote-card"]');
+        });
+        it("player is able to vote for the other player's card only", () => {
+            cy.get('[data-cy="played-cards"] > img').first().then(($img) => {
+                if ($img.hasClass('Cards_allCards__2SdBy')) {
```

Comment:
Just a warning about this kind of thing. These class names can change because they depend on file location (among other things). I'm not saying you shouldn't do this (it's definitely the easiest option) but you may have to update in the future.

Reply:
Yeah, unfortunately there's literally no way you can check if something is a substring of a className in cypress, we'll have to figure out another way in the future :(